pkg/commands/workflows/remove_comments.go

```diff
@@ -71,6 +84,13 @@ func (c *RemovePlanCommentsCommand) Flags() *cli.FlagSet {
 		Usage:   "The GitHub pull request number to remove plan comments from.",
 	})
 
+	f.StringSliceVar(&cli.StringSliceVar{
+		Name:    "comment-type",
```

Comment:
Not sure about this name/API presentation. I dont want to remove all guardian comments because that is wrong. This allows for different types of comments.

Reply:
Alternative names, e.g. `from-actions` `for-actions`

Reply:
Went with `for-command`. Thanks!